### PR TITLE
demo/org-hanging-lines

### PR DESCRIPTION
**Pull request by Xillez**
Added Germanic language tree organization chart demo feat. hanging lines.

---

Asana task: https://app.asana.com/0/1204848917817812/1206408652981595/f

### CODE REVIEW

**Review by MarkusBarstad — APPROVED** [review]
Good job!

**Comment by MarkusBarstad** [comment]
Just 2 things I forgot to check and tell you during the review @Xillez but they are no biggies:

- it would be good to assign yourself to this PR, 
- and add the label `Changelog: Feature` so the changelog will list this new feature

**Review by KacperMadej — CHANGES_REQUESTED** [review]
- [x] By default dataLabels have `useHTML:true1` so tooltip will be under them. Fixable by e.g. `tooltip.outside:true`.
image

- [ ] The series name in the tooltip could be improved (set the series name or edit the tooltip header format to display nothing/something instead of "Series 1")

File: samples/highcharts/demo/organization-hanging/demo.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#container {
+    margin: 1em auto;
+    border: 1px solid silver;
```
[the remark this answers is not in the document]
Comment (Xillez) [reply samples/highcharts/demo/organization-hanging/demo.css:3]: Sorry, kinda forgot to check the css 😅

**Review by KacperMadej — APPROVED** [review]

**Review by TorsteinHonsi — COMMENTED** [review]
Thanks!

But, where's the German language? In other trees it is positioned close to Dutch, then there's the Frisian language close to English. Seems a bit skewed when we list 6 variations of Dutch but no German at all 🇩🇪 

Other than that, the demo is perfect, fitting on mobile and all.

**Review by TorsteinHonsi — APPROVED** [review]
Thanks! Pending review from @marvin19

**Review by marvin19 — APPROVED** [review]
Looks good!

**Pull request merged**